src/features/users/usersSlice.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint no-param-reassign: "error" */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { User } from '../../types/User';
+import { getUsers } from '../../api/users';
+
+type UsersState = {
+  users: User[],
+  error: string,
+};
+
+const initialState: UsersState = {
+  users: [],
+  error: '',
+};
+
+export const getAsyncUsers = createAsyncThunk('users/get', async () => {
+  const users = await getUsers();
+
+  return users;
+});
+
+const usersSlice = createSlice({
+  name: 'users',
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(getAsyncUsers.fulfilled, (state, action) => {
+      state.users = action.payload;
```

Comment:
You have not handled the error state in the reducer for the 'getAsyncUsers' action. If the action fails, there should be a case that sets the error state.